projects/rna/in-place-connection/package.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "jetpack-in-place-connection",
+	"version": "1.0.0",
+	"description": "Jetpack In-Place Connection Component",
+	"main": "index.jsx",
+	"author": "Automattic",
+	"license": "GPL-2.0-or-later",
+	"dependencies": {
+		"@wordpress/i18n": "^3.9.0",
+		"prop-types": "^15.7.2"
+	},
+	"optionalDependencies": {
```

Comment:
I _think_ you might want to use `peerDependencies` here rather than `optionalDependencies`.

Reply:
Thanks!